smart-contracts/advanced/multicall.md

```diff
@@ -0,0 +1,55 @@
+---
+description: >-
+  Multicall allows you to aggregate multiple contract reads into a single JSON-RPC request,
+  and execute multiple state-changing calls in a single transaction on the FVM.
+---
+
+# Multicall
+
+Multicall is a powerful tool that offers several advantages for interacting with smart contracts on the Filecoin Virtual Machine (FVM). 
+
+## Contract address
+
+Multicall has the same, precomputed address for all of the networks it is deployed on. 
+| Name             | Address                                      | Mainnet | Calibration |
+| ---------------- | -------------------------------------------- | ------- | ----------- |
+| [Multicall - Mainnet](https://filfox.info/en/address/0xcA11bde05977b3631167028862bE2a173976CA11?t=3) | `0xcA11bde05977b3631167028862bE2a173976CA11` | ✔️      | ❌          |
+| [Multicall - Calibration](https://calibration.filscan.io/en/address/0xcA11bde05977b3631167028862bE2a173976CA11/) | `0xcA11bde05977b3631167028862bE2a173976CA11` | ❌      | ✔️          |
+
+## Usage
+
+Batching contract reads, one of the most common use cases, allows a single `eth_call` JSON RPC request to return the results of multiple contract function calls. It has many benefits:
```

Comment:
`Batching contract reads` is one of the two main usage. We should introduce both and add one section for `Batch contract write`. For example
## Batching contract reads
Introduction
Example of how to use it.

## Batching contract write
Introduction
Example of how to use it.

Reply:
Implemented contract write section